docs/user_deletion.md

```diff
@@ -0,0 +1,28 @@
+# User Deletion
+
+In it's default setup, deleting a user account revokes their access to the application and soft-deletes the User record. We use the [discard](https://github.com/jhawthorn/discard) gem to soft-delete records across the application.
+
+Depending on legalities of your application, you can choose from three options:
+
+1. Soft-delete(default)
+2. Soft-delete and anonymization
+3. Permanentaly delete the user record
```

Comment:
so for each of these we need to have a heading under named the same so they can look at that section for what they need to do for each. 

otherwise at the moment it seems a bit too all over the place where we need to read the whole doc to see what the user actually needs to do. 

in each of the sections:
1. say they don't need to do anything and this is how ti works
2. mention about the task to setup and what it does and how they should update the script
3. how they would set this up

Reply:
basically the same content that we have below but organised it better so its clear